packages/typescript/src/symbols/reference.ts

```diff
@@ -20,7 +20,7 @@ import { TSModuleScope } from "./ts-module-scope.js";
 import { TSOutputSymbol, TSSymbolFlags } from "./ts-output-symbol.js";
 import { TSPackageScope } from "./ts-package-scope.js";
 
-export function ref(refkey: Refkey): () => string {
+export function ref(refkey: Refkey, type?: boolean): () => string {
```

Comment:
Options bag here would be nice (can imagine other reference customizations that might need to be done, e.g. the selectors tsdoc supports to disambiguate the various parts of a type).